services/checks/checks.go

```diff
@@ -165,13 +166,31 @@ func (s *Service) Run(ctx context.Context) {
 	}
 }
 
-// getChecks returns available checks.
-func (s *Service) getChecks() []check.Check {
+// getChecks returns available MySQL checks.
```

Comment:
```suggestion
// getMySQLChecks returns available MySQL checks.
```

The same thing for methods below.
I wonder why golangci-lint is silent about that.